masdif_override_template.yml

```diff
@@ -0,0 +1,41 @@
+# This file should be used in combination with the docker-compose.yml file of Masdif, if you want to use this
+# project within the Masdif project. This file is not meant to be used on its own.
+#
+# In case you have mounted this project inside the subdirectory `rasa/`, you can start the overall project
+#  by running the following command:
+# cat docker-compose.yml rasa/masdif_override_template.yml | docker-compose -f - up
+#
+# Please make sure, that you have prepared all required environments and initializations beforehand. See the
+# README.md of both projects for more information.
+
+  fuseki-init:
+    # this service is run just for the sake of changing fuseki-data volume
+    # to a non-root user with uid:gid 1000:1000 which is then used inside
+    # fuseki
+    image: openjdk:17-alpine
+    user: root
+    group_add:
+      - '1000'
+    volumes:
+      - fuseki-data:/tmp/fuseki-data
+    command: chown -R 1000:1000 /tmp/fuseki-data
+
+  fuseki:
+    # These final two services are the only ones that need to be built via docker-compose build
+    build:
+      # adapt build context according to your project structure, the following context is used, if you have
+      # mounted this project inside the subdirectory `rasa/`
+      context: rasa/
+      dockerfile: ./docker/fuseki/Dockerfile
+    image: harbour.grammatek.com/sdifi/fuseki-${FUSEKI_VERSION}:${APPLICATION_VERSION:-latest}
```

Comment:
ok, so changing from using $REGISTRY_URL? Will this registry be available for 3rd party users after the project ends?

Reply:
Yes, I think it is more clear to use $REGISTRY_URL